.github/workflows/docker.yaml

```diff
@@ -29,4 +29,4 @@ jobs:
       - name: Run make all inside image
         # javascript fails due to permissions
         # rust fails due to missing stdbool.h
-        run: docker run --rm -v $PWD:/mnt/workspace -t libsbp-build:latest sudo --preserve-env=PATH,RUSTUP_HOME,CARGO_HOME,NVM_DIR,NODE_PATH make c python gen-javascript java docs haskell protobuf gen-rust jsonschema quicktype
+        run: docker run --rm -v $PWD:/mnt/workspace -t libsbp-build:latest sudo --preserve-env=PATH,RUSTUP_HOME,CARGO_HOME,NVM_DIR,NODE_PATH make c python gen-javascript java docs haskell protobuf gen-rust jsonschema quicktype rust
```

Comment:
rust can replace `gen-rust`, and that puts it in the same order as the `all` target.

But not worth another rebuild in CI.  I'll do the javascript one soon, and then this line changes to `make all`

Reply:
Ok, I want to see the current CI build pass, then I'll push this fix-up